thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js

```diff
@@ -1062,12 +1032,10 @@ export default Controller.extend({
      */
     onSubmit() {
       const {
-        metricId,
-        selectedMetric,
         isDuplicateEmail,
         onboardFunctionPayload,
         alertGroupNewRecipient: newEmails
-      } = this.getProperties('metricId', 'selectedMetric', 'isDuplicateEmail', 'onboardFunctionPayload', 'alertGroupNewRecipient');
+      } = this.getProperties('isDuplicateEmail', 'onboardFunctionPayload', 'alertGroupNewRecipient');
```

Comment:
Since we've migrated to a new way of import, let's be consistent about it here.
`getProperties(this, 'isDuplicateEmail',...)`